client/keys/add.go

```diff
@@ -77,7 +78,16 @@ Example:
 	f.Uint32(flagCoinType, sdk.GetConfig().GetCoinType(), "coin type number for HD derivation")
 	f.Uint32(flagAccount, 0, "Account number for HD derivation (less than equal 2147483647)")
 	f.Uint32(flagIndex, 0, "Address index number for HD derivation (less than equal 2147483647)")
-	f.String(flags.FlagKeyAlgorithm, string(hd.Secp256k1Type), "Key signing algorithm to generate keys for")
+	f.String(flags.FlagKeyType, string(hd.Secp256k1Type), "Key signing algorithm to generate keys for")
+
+	// support old flags name for backwards compatibility
+	cmd.PersistentFlags().SetNormalizeFunc(func(f *pflag.FlagSet, name string) pflag.NormalizedName {
```

Comment:
`cmd.PersistentFlags()` should have been `cmd.Flags()` and my fat fingers merged from my phone.
I can fix in on unrelated incoming PR.

Reply:
Fixing here: https://github.com/cosmos/cosmos-sdk/pull/14652/files#diff-ff89a853b81cbba6ecc824fc56f2e0e6e59f56dd17403b5c8e438fb6dad379b5R84 so we are good.